packages/icon-font/package.json

```diff
@@ -14,7 +14,9 @@
         "generate:fonts": "fantasticon ../svg -o ./ -n BrightlayerUIIcons -p blui -t ttf -t woff -t woff2 -t eot -t svg --normalize --font-height 300",
         "build": "yarn clear:fonts && yarn generate:fonts",
         "prettier": "prettier \".md\" --write",
-        "prettier:check": "prettier \"*.md\" --check"
+        "prettier:check": "prettier \"*.md\" --check",
+        "publish:package": "set npm_config_yes=true && npx -p @brightlayer-ui/publish blui-publish",
+        "tag:package": "set npm_config_yes=true && npx -p @brightlayer-ui/tag blui-tag -s blui-icons-svg"
```

Comment:
This is not the icons-svg package (its ust icons...aka icon font)...we should not be tagging it as icons-svg